core/views.py

```
@@ -440,6 +442,16 @@ def form_valid(self: "PostReportView", form: forms.ModelForm) -> HttpResponseRed
         post_report.post = post
         post_report.report_person = report_person
         post_report.save()
+        full_activation_link = self.request.build_absolute_uri(reverse("reported-post", kwargs={"pk": post.id}))
+        mail_status = send_mail(
+            "New reported post",
+            f"A new post has been reported and is awaiting your review here: {full_activation_link}.",
+            settings.DEFAULT_FROM_EMAIL,
+            User.objects.filter(is_superuser=True).values_list("email", flat=True),
+            fail_silently=True,
```

I would say fail silently false. We need to see the error, or have some kind of report. RIght now only the user will see the error, no one else...
For example sentry will not catch it (if implemented) because there is no exception raised. 
It also breaks Zen of Python:
`Errors should never pass silently.`
At least log is required.

---

Hm, yes you are right but I don't know how to do it in correct way, we don't have logger, maybe we should discuss about it and create new epic/ticket

---

just make it fail loud. It's better to fail loud than to hide.